apps/cms-v2/src/modules/providers/payloadcms/payloadcms-client.ts

```diff
@@ -159,12 +170,12 @@ export class PayloadCMSClient {
   }
 
   async upsertProductVariant(context: Context) {
-    this.logger.debug("Trying to upsert product variant");
+    this.logger.debug("updateProductVariant called");
 
     try {
       await this.uploadProductVariant(context);
     } catch (e) {
-      this.logger.debug("Failed to upload, will try to update");
+      this.logger.warn("Failed to upload, will try to update", { error: e });
```

Comment:
question: WDYT

Reply:
its not a warning, this is how upsert works